.github/workflows/cr.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Any Commit
+on:
+  push:
+    branches:
+      - main
+      - v6/environment-api
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  build:
+    if: github.event_name == 'push' || github.event.review.state == 'APPROVED'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - run: corepack enable
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Build
+        run: pnpm build
+
+      - run: pnpx pkg-pr-new publish --compact --pnpm ./packages/vite
```

Comment:
Actually, I forgot @dominikg's feedback. @AmirSa12 we should move to pkg-pr-new being a dev dependency, no?

Reply:
that would be great, yes. Another option would be to use an explicit version here `pnpm dlx pkg-pr-new@1.2.3` 

Also if this only publishes vite, then maybe it's cheaper to run build only in packages/vite?

Reply:
@patak-dev we can do that, we were working on issues so we used latest.

Reply:
@dominikg, Thanks! Yes lets run build only in packages/vite since we only publish that

Reply:
I think we should go with `pnpm dlx pkg-pr-new@0.0` instead of the dev dependency, so we don't get this locked at the patch level in our package lock. This should give us the latest `0.0.x` version IIUC, that would be useful now that the project is releasing bug fixes often.

Reply:
one thing you can add to enhance security even in the event the token is stolen is

https://github.com/sveltejs/vite-plugin-svelte/blob/45a43a4084499559a2e7a08b2403729e835fa06a/.github/workflows/ci.yml#L20

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs


with that, the script not needing the token by default and the fact that we know and trust the org that controls the pkg-pr-new package it could be fine to keep this unpinned if it helps development.

But i kind of wonder if it is a good idea to automate it already if it is still undergoing frequent changes. Wouldn't a comment trigger like with ecosystem-ci be a slower start where you can learn about what works?